### ISSUE TITLE
Add item_effect.inc __item_effect_data array to be readily accessible as json data

### ISSUE THREAD

**Issue by Niphyr** [MechanicalPriest/SimcProfileParser#16]
See #10 for information on previous implementations.

Another file needs to be added from this file: https://github.com/simulationcraft/simc/blob/shadowlands/engine/dbc/generated/item_effect.inc

Structure is an array of `item_effect_t` named `__item_effect_data `:

```c++
static const std::array<item_effect_t, 6712> __item_effect_data { {
```
`item_effect_t` structure:
```c++
struct item_effect_t
{
  unsigned id;
  unsigned spell_id;
  unsigned item_id;
  int8_t   index;
  int8_t   type;
  int16_t  cooldown_group;
  int      cooldown_duration;
  int      cooldown_group_duration;
}
```
The existing class SimcRawItemEffect should be updated to reflect this structure.

**Comment by Niphyr** [comment]
Added as part of https://github.com/MechanicalPriest/SimcProfileParser/commit/7fda159de2da549a68ad92f7389fd32f726a4b3f

**Comment by Niphyr** [comment]
Closed with #22
